.github/workflows/pre-main.yaml

```diff
@@ -155,7 +155,6 @@ jobs:
           working_directory: cnf-certification-test-partner
```

Comment:
general comments here 

I think we should update README, and provide an example in tnf_config.yaml for a helm chart.
Also, It would be helpful to add unit test for the code that parses the helm chart configuration